tests/test_clients.py

```diff
@@ -35,3 +36,43 @@ def test_close_without_clients(hlwm):
 def test_close(hlwm, running_clients_num):
     hlwm.create_clients(running_clients_num)
     hlwm.call('close')
+
+
+@pytest.mark.filterwarnings("ignore:tostring")
+@pytest.mark.parametrize("urgent", [True, False])
+def test_urgent_on_start(hlwm, x11, urgent):
+    # first create a dummy window, such that the second
+    # window is not focused and thus keeps the urgent flag
+    winid_focus, _ = hlwm.create_client()
+    window = x11.create_client(urgent=urgent)
```

Comment:
For added brevity in the tests, maybe you could change `create_client` to return both the `window` and the `windid_str`?

Reply:
Right! This also makes it a bit more consistent with hlwm.create_client()!